app/controllers/facility_accounts_controller.rb

```diff
@@ -158,4 +167,14 @@ def build_account
     ).build
   end
 
+  def set_facility_accounts_for_user
+    order_id = params[:order_id]
+    order = Order.find(order_id) if order_id
+
+    return unless order
```

Comment:
What is the UX in this case?  should we return an empty collection instead?

Reply:
UX was an ugly error in the console 😅 

Fixed now, thanks!